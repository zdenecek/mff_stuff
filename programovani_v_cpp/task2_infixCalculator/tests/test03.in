5
3.5
-3.5
3-2.5
3-3-(-6.2)
2/4
