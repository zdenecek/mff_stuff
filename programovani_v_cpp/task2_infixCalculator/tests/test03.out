3.5
-3.5
0.5
6.2
0
